### ISSUE TITLE
Is it possible to use firefox "inside" CF?

### ISSUE THREAD

**Issue by tuliomenezes** [CommandFusion/iViewer-For-Android#125]
in Android, I cannot see my camera video (mjpeg) in its standard browser. With Firefox for Android, I can see the video.

Inside CF, I also cannot see the video, perhaps because CF uses standard browser. It would be great if there was a way to change CF to use another browser (eg. Firefox)

Camera tested: Foscam FI-8918 (works good in iPad)

Android MID TM7009, V2.3.4, beta build 153
Rikomagic (clone) , V4.0
Motorola Xoom, V4.0

**Comment by fpillet** [comment]
Is Firefox on Android using some kind of plugin to display the video ? What is the format of the streamed video ? Have you tried using a Video object in CF ?

Thanks!

**Comment by tuliomenezes** [comment]
sorry to delay answer, I was out theese days.

I found the error, it was Authentication Method on Video object. In Ipad, I was using HTTP digest, but this option doesn't work in Android. Changed to HTTP Basic and video works good on Android AND iPad/iPhone.

Now I am trying to manage http commands (another thread if I don't succeed)

PS. no plugin in Firefox. Standard browser doesn't show the video from Foscam, BUT iViewer does, with correct authentication method. 

closing this issue!
